src/main/java/com/mercadopago/client/order/OrderClient.java

```diff
@@ -235,5 +233,50 @@ public Order cancel(String orderId, MPRequestOptions requestOptions) throws MPEx
         return order;
     }
 
-   
+    /**
+     * Method responsible for deleting a transaction from the Order
+     *
+     * @param orderId The ID of the order for which the transaction is created
+     * @param transactionId The ID of the transaction to be retrieved
+     * @param requestOptions Metadata to customize the request
+     * @return The response for the action
+     * @throws MPException an error if the request fails
+     * @throws MPApiException an error if the request fails
+     */
+    public OrderTransaction deleteTransaction(String orderId, String transactionId, MPRequestOptions requestOptions)
+            throws MPException, MPApiException {
+        LOGGER.info("Sending order transaction delete request");
+
+        if (StringUtils.isBlank(orderId) || StringUtils.isBlank(transactionId)) {
+            throw new IllegalArgumentException("Order or Transaction id cannot be null or empty");
+        }
+
+        String url = String.format(URL_TRANSACTION_WITH_ID, orderId, transactionId);
+        LOGGER.fine("Delete transaction URL: " + url);
+
+        MPResponse response = send(url, HttpMethod.DELETE, null, null, requestOptions);
+        if (response.getStatusCode() == HttpStatus.NO_CONTENT) {
+            return null;
+        }
+        try {
+            return Serializer.deserializeFromJson(OrderTransaction.class, response.getContent());
+        } catch (Exception e) {
+            LOGGER.severe("Failed to deserialize response: " + e.getMessage());
+            throw new MPApiException("Transaction not found", response);
+        }
```

Comment:
Não há o que deseriliazar, pois o conteúdo é vazio.
Crie um objeto vazio e adicione o response
```suggestion
        OrderTransaction order = new OrderTransaction();
        order.setResponse(response);
```